apps/explorer/tests/utils/localnet.ts

```diff
@@ -38,6 +38,8 @@ export async function split_coin(address: string) {
         },
     });
 
+    await client.waitForTransaction({ digest: result.digest });
```

Comment:
Maybe we could even increase the timeout here, just in case, and make it longer? I think the default is 6s

Reply:
`timeout = 60 * 1000`
It's 60s default no ? Should be fine. 60 is also the playwright test timeout.

Reply:
ahh perfect then, right right